src/views/networkpolicies/list/utils.ts

```diff
@@ -9,3 +11,13 @@ export const getActiveKeyFromPathname = (pathname: string) => {
 
   return TAB_INDEXES.NETWORK;
 };
+
+export const getUrlFromTab = (tabIndex: number | string, namespace: string): string => {
```

Comment:
getNetworkPolicyUrlTab

Reply:
getNetworkPolicyURLTab -- Acronyms should be all uppercase.